DEVELOPMENT.md

```diff
@@ -0,0 +1,100 @@
+# Litmusctl Local Development Setup Guide
+
+## Introduction
+
+Welcome to the local development setup guide for **`litmusctl`**. This guide will walk you through the steps required to set up and run **`litmusctl`** on your local machine.
+
+## Important Note
+
+Before running **`litmusctl`**, make sure you have a Chaos Centre running. Ensure that the Chaos Centre version is compatible with the **`litmusctl`** version you are using.
+
+## Prerequisites
+
+Before you begin, ensure that you have the following prerequisites installed on your machine:
+
+- [Go programming language](https://golang.org/doc/install) (version or later)
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- Kubeconfig - `litmusctl` needs the kubeconfig of the k8s cluster where we need to connect litmus Chaos Delegates. The CLI currently uses the default path of kubeconfig i.e. `~/.kube/config`.
+
+## Clone the Repository
+
+```bash
+git clone https://github.com/litmuschaos/litmusctl.git
+
+cd litmusctl
+```
+
+## **Install Dependencies**
+
+```bash
+go mod download
+```
+
+## **Configuration**
+
+Before running **`litmusctl`**, update the following configuration paths in the **`pkg/utils/constants.go`**
+
+From this
+
+```go
+// Graphql server API path
+GQLAPIPath = "/api/query"
+
+// Auth server API path
+AuthAPIPath = "/auth"
+```
+
+To this
+
+```go
+// Graphql server API path
+GQLAPIPath = "/query"
+
+// Auth server API path
+AuthAPIPath = ""
+```
+
+Inside pkg/apis/upgrade.go change this
+
+```go
+resp, err := SendRequest(SendRequestParams{Endpoint: cred.Endpoint + utils.GQLAPIPath, Token: cred.Token}, []byte(query), string(types.Post))
+```
+
+To this
+
+```go
+// Query to fetch Infra details from server
+	resp, err := SendRequest(SendRequestParams{Endpoint: "http://localhost:8080/query", Token: cred.Token}, []byte(query), string(types.Post))
```

Comment:
@deep-poharkar with the new server-endpoint addition in litmusconfig this is not required

Reply:
yes, my bad. updated it